tests/Rector/General/ConvertTypo3ConfVars/config/configured_rule.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+use Ssch\TYPO3Rector\Rector\General\ConvertTypo3ConfVarsRector;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import(__DIR__ . '/../../../../../config/services.php');
+    $services = $containerConfigurator->services();
+
+    $services->set(ConvertTypo3ConfVarsRector::class);
+};
```

Comment:
This is the biggest change for good, with Rector testing.

Instead of magically loading root `rector.php` and combining it with `setParameter()` in test case + `getRectorClass()`, this is now 1:1 identical to normal Rector use.

- provide config
- configure wwat you need
- run the tests